bcipy/simulator/demo/button_press_model.py

```diff
@@ -0,0 +1,78 @@
+"""Signal model for button presses."""
+from pathlib import Path
+from typing import List
```

Comment:
This can go to the signal module. I think this is a nicer implementation of what's happening than simply using the `compute_probs_after_preview`